### DISCUSSION TITLE
Hi, is it possible to configure VPS through Vite config to bundle into SPA mode? The expected output after bundling is an index.html. The current setup is a Node.js server-side project.

### DISCUSSION THREAD

**Discussion by algesthesiah** [vikejs/vike#1280] [Discussion]
only need spa ,not ssg

**Comment by algesthesiah** [comment]
The file routing feature of VPS is powerful, and I want to leverage this functionality in conjunction with SPA. The current situation with VPS requires configuring SPA for each page individually.

**Reply by brillout** [reply]
Again, please respect https://brillout.github.io/rules.
